structuralcodes/geometry/_geometry.py

```diff
@@ -527,6 +547,7 @@ def __init__(
                 for pg in self.point_geometries
             ]
             self.geom = MultiPolygon(geoms_representation)
+        self._reinforced_concrete = None
```

Comment:
Remember to define this attr in the class header.

Reply:
Sorry, maybe I am misunderstanding the comment: what do you mean to define this in the class header? I think you are referring to add it before the __init__ right after the class definition. I am not understanding exactly why I should do that... Sorry for the for sure silly question..

Reply:
That is what I meant 😃 good to have it defined and typed one place, and with a reasonable default value.